src/Variables.ts

```diff
@@ -423,7 +424,12 @@ export const Globals: GlobalVariables = {
 
   historyCountMax: 20,
 
-  isEvent: false,
+  get isEvent () {
+    return Events.list.active.length > 0
+  },
+
+  set isEvent (_: boolean) {},
```

Comment:
funny hack because we do `Object.assign(Globals, { ...blankGlobals })` which can't be done on a readonly property